From a77e9518cb87e1a5e3f22d9233dbeac678b1fc19 Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Tue, 14 Aug 2018 17:22:49 +0800
Subject: [PATCH 4379/5242] MLK-19217 ARM64: dts: 8qm lpddr4: remove flexcan
 node from domu

commit  22dc9913ab1fc595fe12885b6c02b0012dc42168 from
https://source.codeaurora.org/external/imx/linux-imx.git

Fixes bd21312fe706("MLK-19205 ARM64: dts: 8qm arm2: add dom0/domu device tree")
Remove flexcan node from DomU, flexcan node not exists in dtsi now.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts  |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts
index ab687ff1..dba80e3 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts
@@ -361,10 +361,6 @@
 	status = "okay";
 };
 
-/delete-node/ &flexcan1;
-/delete-node/ &flexcan2;
-/delete-node/ &flexcan3;
-
 &i2c1_lvds1 {
 	reg = <0x1 0x57247000 0x0 0x1000>;
 	status = "okay";
-- 
1.7.9.5

